app/renderer/components/Inspector/Inspector.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import { debounce } from 'lodash';
```

Comment:
I thought we usually do `import _ from 'lodash'`

Reply:
elsewhere in AD we do it this way, so i'm just following suit.